kijiji_repost_headless/kijiji_api.py

```diff
@@ -115,6 +116,16 @@ def login(self, username, password):
         }
         api_url = 'https://www.kijiji.ca/anvil/api'  # API endpoint
         resp = self.session.post(api_url, json=payload)
+        """
+        
+        dirs = __file__.split("/")
+        ssidDir = ""
+        for i in range(len(dirs)-2):
+            ssidDir += dirs[i] + "/"
+        ssidDir += "ssid.txt"
+        ssidFile = open(ssidDir, "r")
```

Comment:
Dangling file handle; you do not close the file after opening it.

Use the `with` statement which will automatically close the file handle after the with block exits.